subcommands/targets/tag.go

```diff
@@ -78,6 +78,8 @@ func doTag(cmd *cobra.Command, args []string) {
 					targetTags := tags
 					if tagAppend {
 						targetTags = Set(custom.Tags, tags)
+					} else if len(targetTags) == 0 {
+						subcommands.DieNotNil(fmt.Errorf("Cannot change tags from %s -> %s. Empty values are not allowed.", custom.Tags, targetTags))
```

Comment:
I'm wondering why do we make this check here?
Wouldn't it be easier and better to check `if len(tags) == 0 { errorOut }` at line 60?
Or maybe just set something like the minlength in cobra argument definition?